pretyping/evarsolve.ml

```diff
@@ -564,8 +564,8 @@ let alias_distinct l =
   check (Int.Set.empty, Id.Set.empty) l
 
 let get_actual_deps env evd aliases l t =
-  if occur_meta_or_existential evd t then
-    (* Probably no restrictions on allowed vars in presence of evars *)
+  if occur_meta evd t then
```

Comment:
This is a semantic change isn't it?

Reply:
A minor one but yes. I'll put it another PR.